library/oeUI/src/OemrUI.php

```diff
@@ -0,0 +1,278 @@
+<?php
+/**
+ * OemrUI class.
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author Ranganath Pathak <pathak@scrs1.org>
+ * @copyright Copyright (c) 2018 Ranganath Pathak <pathak@scrs1.org>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+namespace OpenEMR\OeUI;
+
+use OpenEMR\Core\Header;
+
+Header::setupHeader(['jquery-ui', 'jquery-ui-base', 'no_jquery', 'no_bootstrap', 'no_fontawesome', 'no_main-theme', 'no_textformat', 'no_dialog' ]);
```

Comment:
This is unusual to do; ie. having a class run code without using the class. It can also lead to duplicate including of js packages. Would is make more sense to make these packages required on scripts that call these classes?

Reply:
This is setup to only add` 'jquery-ui', 'jquery-ui-base` the rest are excluded. The issue  becomes complicated when you have to remember to add these meta tags on each page.  I agree that it will cause duplicate jquery ui tags unless by default you want all pages to load these tags